src/test/php/lang/reflection/unittest/PropertyHooksTest.class.php

```diff
@@ -0,0 +1,59 @@
+<?php namespace lang\reflection\unittest;
+
+use ReflectionProperty;
+use lang\reflection\AccessingFailed;
+use test\verify\Condition;
+use test\{Assert, Expect, Test};
+
+#[Condition(assert: 'method_exists(ReflectionProperty::class, "getHooks")')]
```

Comment:
We cannot yet use `PHP_VERSION_ID` here as https://github.com/php/php-src/pull/13455 is yet to be merged.